test/fixtures/users.yml

```diff
@@ -13,6 +13,7 @@ state1_enroller:
   authy_enforced: false
   api_enabled: false
   role: 'enroller'
+  notes: ''
```

Comment:
I don't think this is necessary unless we're giving it a value and testing/verifying this in the system tests, which I think is a great idea. You can add some checks here: https://github.com/SaraAlert/SaraAlert/blob/master/test/system/roles/admin/dashboard/dashboard_verifier.rb